src/scores/continuous/correlation/__init__.py

```diff
@@ -0,0 +1,7 @@
+"""
+Import the functions from the implementations into the public API
+"""
+
+from scores.continuous.correlation.correlation_impl import pearsonr
+
+__all__ = ["pearsonr"]
```

Comment:
This is consistent with how R does it, but do we typically use 'r' as a variable for correlation? I'm just as happy with just 'pearson'. I don't think users are likely to mind either way, so this is up to you, I just wanted to mention it.

Reply:
Yes, $r$ is generally used. See the "For a sample" section in https://en.wikipedia.org/wiki/Pearson_correlation_coefficient

It's also consistent with python implementations (e.g., scipy)

Reply:
That's fine, thanks.